about.html

```diff
@@ -0,0 +1,161 @@
+          <a href="index.html"><img src="./image/logo.jpg" alt="LOGO" class="logo"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Securam Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">Securam Entertainment</h1>
+      <div class="first-section-para">
+        <p>
+          The leading entertainment platform in Nigeria, unifying the music industry in other to stand firm in global entertainment. This is the very first show bringing the leading artists in Africa together shearing their challenges and how we can continue to put Nigeria in the global space.
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions.
+        </p>
+        <p class="email-address">
+          securamesec@gmail.com
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="flex-content">
+      <h2> NIGERIA MUSIC SUMMIT 2021</h2>
+      <hr>
+      <p class="mobile-para">
+        Celebration of Nigerian hiphop miusic, its global reception, and challenges. creating a positive change in Nigerian music industry.
+        2021.12.16(THU) ~ 17(FRI)
+      </p>
+      <p class='desktop-para'>Celebration of Nigerian hiphop miusic, its global reception, and challenges. creating a positive change in Nigerian music industry.
+        2021.12.16(THU) ~ 17(FRI)</p>
+      <img src="./image/logo.jpg" alt="logo" class="logo">
+    </div>
+    <hr class="horizontal-line">
+      
+  </section>
+
+  <section class="past-section">
+    <div class="past-section-container flex-past">
+      <h2>Past Music Concerts</h2>
+      <hr>
+      <p>Take a look at the past music Concerts hosted by our leading artists.</p>
+      <div class="pics">
+      <img src="./image/wizkidO2.jpg" alt="Wizkid soldout O2 Arena ">
+      <img src="./image/burnaboyO2.jpg" alt="Burnaboy soldout O2 Arena">
+    </div>
+    </div>
+  </section>
```

Comment:
- [ ] When it comes to the mobile section, make sure these images have the same height or style. I would recommend making the same height as the second image has.

![html13](https://user-images.githubusercontent.com/55635977/142590960-91a467ea-9a9a-4b06-9167-49c9aedee136.png)